jetcd-core/src/main/java/io/etcd/jetcd/ClientConnectionManager.java

```diff
@@ -225,7 +225,7 @@ void close() {
                 "%s://%s/%s",
                 IPNameResolver.SCHEME,
                 builder.authority() != null ? builder.authority() : "",
-                endpoints.stream().map(e -> e.getHost() + ":" + e.getPort()).collect(Collectors.joining(",")));
+                endpoints.stream().map(URI::toString).collect(Collectors.joining(",")));
```

Comment:
Not very sure it is needed: the authority is defined as`[userinfo@]host[:port]` but given that the `userinfo` do not make any sense here, why do we need toString ? toString also included the scheme which does not make any sense in the ip resolver

Reply:
Use `URI::toString` then we can  use `URI::create` in another place. it is like serialize and deserialize, we just use URI as a basic tool to reduce hard code

Reply:
yes but the IP resolver is not expecting a full URI, just host and port

Reply:
all right, we can aslo use host and port to create URI just like  `URI.create(String.format("ip://%s", address))`